.env.example

```
@@ -102,9 +107,31 @@ STRIPE_WEBHOOK_SECRET=
 
+# ---- Billing operations ---------------------------------------------------
+# See docs/operations/billing-runbook.md for alert thresholds and procedures.
+# Assumed variable cost per connected minute. Used to record a provider cost
+# estimate before the provider reports actual usage, so margin dashboards are
+# never blank. Raising this lowers reported margin; it does not change what a
+# customer is charged.
+BILLING_VARIABLE_COST_RESERVE_USD_PER_MINUTE=0.12
+# How often the reconciliation worker runs. Requires WORKERS_ENABLED=true on at
+# least one instance, otherwise balance drift and leaked reservations are never
+# repaired.
+BILLING_RECONCILIATION_CRON="*/15 * * * *"
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BILLING_RECONCILIATION_CRON key should go before the BILLING_VARIABLE_COST_RESERVE_USD_PER_MINUTE key

(UnorderedKey)